src/frontend/admin/src/components/templates/products/form/sections/main/ProductFormMain.tsx

```diff
@@ -94,6 +98,14 @@ export function ProductFormMain({
     methods.setValue("type", productType);
   }, [productType]);
 
+  useEffect(() => {
```

Comment:
I found a bug, when I click on submit the warning modal is shown:


https://github.com/openfun/joanie/assets/9628870/e3c4c702-ace7-4006-854f-74b7f418f9e4